specification/sql/resource-manager/Microsoft.Sql/preview/2020-11-01-preview/examples/ListOperations.json

```diff
@@ -0,0 +1,10 @@
+{
+  "parameters": {
+    "api-version": "2020-11-01-preview"
+  },
```

Comment:
do we know why we need to add this example file to 2020-11-01-preview? 

all the swagger/example files in SQL are auto-generated from DSMainDev repo since 2020-02-02-preview. Manually updating would break our auto-pipeline. It means this example file needs to update for all the versions after 2020-02-02-preview.

Reply:
I find 2020-11-01-preview in default tag

```
``` yaml $(tag) == 'package-composite-v5'
input-file:
- Microsoft.Sql/stable/2014-04-01-legacy/backups_legacy.json
- Microsoft.Sql/stable/2014-04-01/connectionPolicies.json
- Microsoft.Sql/stable/2014-04-01/dataMasking.json
- Microsoft.Sql/stable/2014-04-01/geoBackupPolicies.json
- Microsoft.Sql/stable/2014-04-01/metrics.json
- Microsoft.Sql/stable/2014-04-01-legacy/replicationLinks_legacy.json
- Microsoft.Sql/stable/2014-04-01/serverCommunicationLinks.json
- Microsoft.Sql/stable/2014-04-01/serviceObjectives.json
- Microsoft.Sql/stable/2014-04-01-legacy/sql.core_legacy.json
- Microsoft.Sql/stable/2014-04-01-legacy/usages_legacy.json
- ./Microsoft.Sql/preview/2020-11-01-preview/BlobAuditing.json
- ./Microsoft.Sql/preview/2020-11-01-preview/DatabaseAdvisors.json
- ./Microsoft.Sql/preview/2020-11-01-preview/DatabaseAutomaticTuning.json
- ./Microsoft.Sql/preview/2020-11-01-preview/DatabaseColumns.json
- ./Microsoft.Sql/preview/2020-11-01-preview/DatabaseRecommendedActions.json
```

if these are auto-generated, maybe i should revert them